watcher_test.go

```diff
@@ -27,8 +37,8 @@ func initWatcherWithOptions(t *testing.T, wo WatcherOptions) (*casbin.Enforcer,
 	return e, w.(*Watcher)
 }
 
-func initWatcher(t *testing.T) (*casbin.Enforcer, *Watcher) {
-	return initWatcherWithOptions(t, WatcherOptions{})
+func initWatcher(t *testing.T, cluster ...bool) (*casbin.Enforcer, *Watcher) {
```

Comment:
In order to merge this PR, you need to test the function of cluster. But there's no real test here. It is not used